client/task-list/tasks/payments/RemotePayments/components/PaymentConnect.js

```diff
@@ -103,39 +140,49 @@ export const PaymentConnect = ( {
 		},
 	} );
 
-	return (
-		<Form
-			initialValues={ getInitialConfigValues() }
+	const DefaultSettings = ( props ) => (
+		<SettingsForm
+			fields={ fields }
+			isBusy={ isOptionsRequesting }
 			onSubmitCallback={ updateSettings }
```

Comment:
I'm still seeing this as `onSubmitCallback`- maybe the changes weren't pushed?

Reply:
Good catch, I updated the component but not it's usage 👍🏻